Resources/views/admin/fields/file-link-multiple.blade.php

```diff
@@ -1,54 +1,21 @@
 <script src="{{ Module::asset('dashboard:vendor/jquery-ui/jquery-ui.min.js') }}"></script>
```

Comment:
Think it's not needed for squash here as it's just an improvement and not a 'fix mistake' - type commit.